### PR TITLE
Update Blockchain.thy

### PR DESCRIPTION
**Pull request by tirix**

### CODE REVIEW

**Comment by diekmann** [comment]
LGTM. Travis is also red which is the color of love. Great!

**Comment by tirix** [comment]
It's a pleasure to contribute to that ambitious project. 
Wishing you good luck for future developments!

File: Blockchain.thy [hunk 1]
```diff
@@ -4,5 +4,6 @@ begin
 datatype 'a blockchain = Empty | Block "'a blockchain"
   
 (*User Guide: Add your blocks here*)
+value \<open>Block (Block (Block (Block (Block Empty))))\<close>
```
Comment (larsrh) [inline Blockchain.thy:7]: Is this a blockchain split? :crying_cat_face:

**Comment by diekmann** [comment]
Worry not @larsrh, for our next-gen blockchain technology can resolve the split.

**Pull request merged**